mars/optimizes/core.py

```diff
@@ -15,24 +15,37 @@
 # limitations under the License.
 
 from ..tensor.fuse.ne import NUMEXPR_INSTALLED
+from ..tensor.fuse.jx import JAX_INSTALLED
 from .ne import NeOptimizer
 from .cp import CpOptimizer
+from .jx import JaxOptimizer
 
+engine_dic = {'numexpr': NeOptimizer,
+              'cupy': CpOptimizer, 'jax': JaxOptimizer}
 
-class Optimizer(object):
-    engine_dic = {'numexpr': NeOptimizer,
-                  'cupy': CpOptimizer}
 
-    def __init__(self, graph, engine=None):
-        self._graph = graph
-        if not engine:
-            self._engine = 'numexpr' if NUMEXPR_INSTALLED else 'numpy'
-        else:
-            self._engine = engine
+def optimize(graph, engines, keys=None):
+    if engines is None:
+        # the sequence of optimize
+        engines = []
```

Comment:
I wonder if we can add a ``default_engines`` list and build it outside the function as installations do not change once the program starts?